src/local/Message.ts

```diff
@@ -0,0 +1,16 @@
+export default class Message {
+  private isAcknowledged?: boolean;
+  private readonly ack?: (reason: string | null, isAcknowledged: boolean) => void;
```

Comment:
If `ack` is `private readonly`, but never assigned, it will never exist. When is `ack` supposed to be set?

Reply:
The entire Message class could probably be removed, it isn't used anywhere or even included in the root index export

Reply:
Remove it then.